client/system/info_windows.go

```diff
@@ -93,3 +138,33 @@ func getBuildVersion() string {
 	ver := fmt.Sprintf("%d.%d.%s.%d", major, minor, build, ubr)
 	return ver
 }
+
+func sysNumber() (string, error) {
+	var dst []Win32_BIOS
+	query := wmi.CreateQuery(&dst, "")
```

Comment:
When I was building it it was complaining that wmi is undefined because the import is missing

Reply:
This is strange because in the imports the "github.com/yusufpapurcu/wmi" is there